### PR TITLE
Fix link

### PR DESCRIPTION
**Pull request by mminer237**
#### Summary
Fixes link to correct location of FontFace page

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [✅ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/fontfaceset/load/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ load(font, text)
 
 ### Return value
 
-A {{jsxref("Promise")}} of an {{jsxref("Array")}} of {{jsxref("FontFace")}} loaded. The
+A {{jsxref("Promise")}} of an {{jsxref("Array")}} of {{domxref("FontFace")}} loaded. The
```
Comment (wbamberg) [inline files/en-us/web/api/fontfaceset/load/index.md:34]:
```suggestion
A {{jsxref("Promise")}} fulfilled with an {{jsxref("Array")}} of loaded {{domxref("FontFace")}} objects. The
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @mminer237 , nice catch!

**Review by Lexxi5 — COMMENTED** [review]
domxref("FontFace

**Review by Lexxi5 — COMMENTED** [review]
https://github.com/mdn/content/pull/15926/files/624e0a8f1047e506cfe240bfad431fae75313be2#diff-5167501368204985b14e3f9d7fd20437218652f4d8b7f4a26038cde4d9b7f678

**Review by Lexxi5 — COMMENTED** [review]
@@ -31,7 +31,7 @@ load(font, text)

**Pull request merged**